apps/site/next-data/downloadSnippets.ts

```diff
@@ -24,9 +26,10 @@ const getDownloadSnippets = (lang: string): Promise<Array<DownloadSnippet>> => {
   // the data directly within the current thread, which will anyways be loaded only once
   // We use lazy-imports to prevent `provideBlogData` from executing on import
   if (ENABLE_STATIC_EXPORT || IS_NOT_VERCEL_RUNTIME_ENV) {
-    return import('@/next-data/providers/downloadSnippets').then(
-      ({ default: provideDownloadSnippets }) => provideDownloadSnippets(lang)!
+    const { default: provideDownloadSnippets } = await import(
+      '@/next-data/providers/downloadSnippets'
     );
```

Comment:
same comment here - this isn't just style, it's cleaner in the case where it fails to debug the problem.